test/cmocka/src/math/arithmetic/base_10_logarithm.c

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+//
+//
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <math.h>
+#include <cmocka.h>
+#include <string.h>
+
+#include <sof/math/log.h>
+#include <sof/audio/format.h>
+#include <sof/string.h>
+#include <sof/common.h>
+#include "log2_tables.h"
+/* 'Error[max] = 0.0000071204522865,THD = -102.9498483876767807 ' */
+#define CMP_TOLERANCE 0.0000071204522865
```

Comment:
Is this some manually found tolerance that just passes? Add a comment to magic number why use use such, the above -102.9498... comment looks very magic too :)

Reply:
Done